exercise-2/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Exercise2</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <h2 style="margin-left: 200px; padding-top: 30px">
+      Why LocoNav's Vehicle Tracking System?
+    </h2>
+
+    <div class="first">
+      <img src="img/first.png" alt="INTELLIGENT VEHICLE TRACKING" />
+      <p>INTELLIGENT VEHICLE TRACKING</p>
+      <p class="trans">
+        Whether you own a single vehicle or are a <a href="">fleet owner </a>,
```

Comment:
The no. of lines / words per line in given design vs your design should be **exactly same**.